stacker/plan.py

```diff
@@ -197,11 +209,23 @@ def _single_run(self):
         """Executes a single run through the plan, touching each step."""
         for step_name, step in self.list_pending():
             waiting_on = []
+            failed = False
             for required_stack in step.requires:
+                if self[required_stack].failed:
+                    logger.debug(
+                        'Failing stack \"%s\" due to failed dependency '
+                        '\"%s\"', step_name, required_stack)
+                    step.set_status(FailedStatus("dependency has failed"))
```

Comment:
think it's worth identifying the dependency here?

Reply:
The listing of failed stacks at the end should be enough to figure out what failed. But being explicit can't hurt.

Reply:
Updated the message and moved it to `warn` level so it's always seen.